### ISSUE TITLE
Accessibility labels are different in iOS 8.3 / Xcode 6.3

### ISSUE THREAD

**Issue by jshier** [kif-framework/KIF#623]
I'm trying to narrow down the cause of a change in iOS 8.3 / Xcode 6.3 that has broken quite a few of our accessibility labels and therefore KIF tests. It seems that multiline labels, which used to have the accessibility label "First Line Second Line" now have the label "First Line\nSecond Line". I'm not sure whether this is an SDK change or KIF not handling the new version appropriately. I did see that KIF tries to filter newlines from accessibility labels, but I haven't looked closely enough to see if that logic now fails on 8.3. I'll report more as I investigate, but I wanted to report it now in case it was something that had cropped up before.

**Comment by bnickel** [comment]
KIF has logic that is supposed to handle this. When it compares labels it calls [`StringsMatchExceptLineBreaks`](https://github.com/kif-framework/KIF/blob/master/Additions/UIView-KIFAdditions.m#L43).  Looks like I only did the checking if the test expects a line break but the label doesn't have one (the original break).  Does your test run fine in 8.2 and 8.3 if you search for "First Line\nSecond Line"?

**Comment by jshier** [comment]
I don't believe so, as the output from the failed to find element error doesn't list any with newlines under 8.2. Under 8.3, that listing does show the newlines and therefore won't match any of our newline-less strings.

**Comment by phatmann** [comment]
Fixed via #643. @jshier if you are still running into problems, please re-open this issue.

**Comment by jshier** [comment]
@phatmann Can you cut a new pod version?

**Comment by phatmann** [comment]
New version (3.2.2) pushed.
